docs/source/03_tutorial/06_visualise_pipeline.md

```diff
@@ -96,3 +96,68 @@ kedro viz --load-file my_shareable_pipeline.json
 ```
 
 And this will visualise the pipeline visualisation saved as `my_shareable_pipeline.json`.
+
+
+## Visualise plotly-express charts in Kedro-viz
+
+You can view plotly-express charts in Kedro-Viz when you use Kedro's plotly data connector.
+
+There are two plotly data connectors in Kedro :
+- [plotly.PlotlyDataSet](https://kedro.readthedocs.io/en/stable/kedro.extras.datasets.plotly.PlotlyDataSet.html#kedro.extras.datasets.plotly.PlotlyDataSet) - This is where you specify the configurations of your plot in the `catalog.yml`.
+
+```yaml
+bar_plot:
+    type: plotly.PlotlyDataSet
+    filepath: data/08_reporting/bar_plot.json
+    plotly_args:
+        type: bar
+        fig:
+            x: features
+            y: importance
+            orientation: h
+        layout:
+            xaxis_title: x
+            yaxis_title: y
+            title: Test
+```
+
+
+- [plotly.JSONDataSet](https://kedro.readthedocs.io/en/stable/kedro.extras.datasets.plotly.JSONDataSet.html#kedro.extras.datasets.plotly.JSONDataSet) - This is where you specify the configurations of your plot using plotly-express python library.
+
+```python
+import plotly.express as px
+from kedro.extras.datasets.plotly import JSONDataSet
+
+
+def plot():
+    fig = px.bar(x=["a", "b", "c"], y=[1, 3, 2])
+    data_set = JSONDataSet(filepath="test.json")
+    data_set.save(fig)
+    reloaded = data_set.load()
+    assert fig == reloaded
+```
+
+For plotly.JSONDataSet, you will also need to specify the output type in `catalog.yml`.
+
+```yaml
+test.json:
+  type: plotly.JSONDataSet
+  filepath: data/08_reporting/bar_plot.json
+```
+
+Once the above set-up is completed, you can do a `kedro run` followed by `kedro viz` and your kedro-viz pipeline will show a new dataset type with icon ![](../meta/images/icon-image-dataset.svg) . Once you click on the node, you can see a small preview of your plotly chart in the metadata panel.
+
+![](../meta/images/pipeline_visualisation_plotly.png)
+
+
+
+The visualisation can be expanded using the 'Expand Plotly Visualisation' button.
+
+
+![](../meta/images/pipeline_visualisation_plotly_expand.png)
+
+## Visualise metrics from your experiments
```

Comment:
I have a thought that this entire section should be placed in the "Experiment Tracking" documentation. But I would still mention that this is possible by including a sentence in this Plotly documentation about "We've also used this Plotly integration to allow you to visualise metrics from your experiments - making it easier to view trends over time." I'd put this sentence in the introduction and hyperlink to where you put this section.

Reply:
While on this topic, dropping by a quick note that this will be mentioned in the experiment tracking docs that I am currently drafting as well - we can come back to further add the reference once it's merged in